README.md

```diff
@@ -39,6 +39,8 @@ Usage: chart_test.sh <options>
     --verbose         Display verbose output
     --no-lint         Skip chart linting
     --no-install      Skip chart installation
+    --force           Force charts linting/installation
+    --chart           Lint/install a standalone chart (ignored if --force is used)
```

Comment:
I have a couple questions:

1. What is the case for testing all the charts compared to a list of charts? Would it be enough to pass a list of charts to test in even if they don't have changes? With a list it could be anywhere from one to all of them?
1. Instead of force, should it be `--check-all-charts` to be more descriptive?
1. Is there an underlying use case or example that might inform what the input should look like?

Reply:
ok, an example of forcing all charts, is testing them locally, specially when it is used for your helm charts git repo. I was partially asked by Jfrog developers to have such option, now they test charts, on any cluster they want with the simple command `make mac -- --force --no-install` or `make mac -- --force --no-lint`.
https://github.com/jfrog/charts/blob/master/Makefile#L23

I found `--check-all-charts` a bit too long if you need to type it regularly :)

Also a new PR introducing `gke` option for manual testing on remote clusters https://github.com/jfrog/charts/pull/36
As there are many cases of charts testing outside the CI too.

Reply:
what about `--force` -> `--charts`?
one chart `--chart`
all charts `--charts`
flags are humans, env vars for CI :)

Reply:
If you're already using make you can just create a short command that incorporates `--check-all-charts`. Though, we could shorten it to `--all-charts` and still be descriptive.

How about an `--all-charts` and single chart `--chart` option for starters. Thought, I will want to chat with @unguiculus before moving on this.

Reply:
As for gke, we should keep it to a separate issue. In general, we do not want to support all cloud providers here or prioritize any of them. The CNCF, which Helm is part of, strives to be vendor neutral. Our stance has been that the image should be extended for a specific cloud provider. For example, we do that on the Charts repo... https://github.com/helm/charts/tree/master/test

Reply:
`gke` is our another example use case at jfrog :) I just gave you a few use cases